src/EnergyPlus/HVACManager.cc

```diff
@@ -891,7 +891,7 @@ namespace HVACManager {
 							}
 						} // humidity ratio not converged
 
-						if ( AirLoopConvergence( AirSysNum ).HVACTempNotConverged ) {
+						if( any( AirLoopConvergence( AirSysNum ).HVACTempNotConverged ) ) {
```

Comment:
@Myoldmopar if you already have an array of booleans calling the `any` function is fine wrt performance. The scenario I addressed for performance is when you are creating a temporary array to then just call any on and discard -- very inefficient. That would be something like `if ( any( A > B ) ) ...` and for those I added some special methods to avoid the inefficiency so we could write it as `if ( any_gt( A, B ) ) ...` and I changed all instances in the code I found. This is definitely something to keep an eye out for in new code.